src/Uno.UI/UI/Xaml/Media/IGradientBrush.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.UI.Xaml.Media;
+
+namespace Windows.UI.Xaml.Media
+{
+	/// <summary>
+	/// This interface defines common functionality between <see cref="GradientBrush"/> and <see cref="RadialGradientBrush"/>
+	/// since, in UWP, <see cref="RadialGradientBrush"/> doesn't derive from <see cref="GradientBrush"/>
+	/// </summary>
+	internal partial interface IGradientBrush
+	{
+	}
+}
```

Comment:
Personally here I would prefer to define a private type `_Layer` per platform and use it in a single file interface, I find this much more readable / maintainable / converging:

```suggestion

#if __IOS__ || __MACOS__
using _Layer = UIKit.CALayer;
#elif __ANDROID__
using _Layer = Shader;
#elif __WASM__
using _Layer = SvgElement;
#endif

﻿using Microsoft.UI.Xaml.Media;

namespace Windows.UI.Xaml.Media
{
	/// <summary>
	/// This interface defines common functionality between <see cref="GradientBrush"/> and <see cref="RadialGradientBrush"/>
	/// since, in UWP, <see cref="RadialGradientBrush"/> doesn't derive from <see cref="GradientBrush"/>
	/// </summary>
	internal partial interface IGradientBrush
	{
		_Shader GetShader(Windows.Foundation.Rect destinationRect);
	}
}
```

Thought @jeromelaban , @carldebilly @davidjohnoliver ?

Reply:
@dr1rrb I wanted the method name to match the terminology of each platform. But open to change that.

cc @jeromelaban @carldebilly @davidjohnoliver for opinions.